connectors/sources/github.py

```diff
@@ -659,7 +668,9 @@ def get_rate_limit_encountered(self, status_code, message):
 
     async def _get_retry_after(self, resource_type):
         current_time = time.time()
-        response = await self._get_client.getitem("/rate_limit")
+        response = await self._get_client.getitem(
+            "/rate_limit", oauth_token=self._access_token()
+        )
```

Comment:
This naming always confused me, `oauth_token` is just api key in essence?

Reply:
This is the term used by `gidgethub`, to differentiate from `jwt`. `gidgethub` [uses different headers](https://github.com/gidgethub/gidgethub/blob/7f8cec45922056a38c51ece276844a4bc028e6f5/gidgethub/sansio. Bearer`. See [here](https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28#about-authentication):
>**Note**: In most cases, you can use `Authorization: Bearer` or `Authorization: token` to pass a token. However, if you are passing a JSON web token (JWT), you must use `Authorization: Bearer`.